Source/DafnyLanguageServer.Test/Various/CancelVerificationTest.cs

```diff
@@ -80,7 +80,7 @@ public class CancelVerificationTest : ClientBasedLanguageServerTest {
     Ack(m - 1, Ack(m, n - 1))
 }
 
-method {:timeLimit 10} test() {
+method {:rlimit 10000} test() {
```

Comment:
I get that with the legacy CLI we only have `/rlimit` as an option, but for attributes we should only use the non-deprecated `{:resource_limit}` attribute IMO.